mobile-app/cypress/e2e/functional/wallet/loans/loans.spec.ts

```diff
@@ -17,6 +17,21 @@ function addCollateral(): void {
   cy.getByTestID("vault_card_0_total_collateral_amount").contains("$1,500.00");
 }
 
+function checkTokensSortingOrder(
+  sortedType: string,
+  firstToken: string,
+  lastToken: string
+): void {
+  const containerTestID = '[data-testid="loan_screen_token_lists"]';
+  const sortButtonTestID = "loans_tokens_sort_toggle";
+  cy.getByTestID(sortButtonTestID).click();
+  cy.getByTestID(`select_sort_${sortedType}`).click();
+  cy.wait(3000);
+  cy.getByTestID(sortButtonTestID).contains(sortedType);
+  cy.get(containerTestID).children().first().contains(firstToken);
```

Comment:
Good to have if we could also check if the actual sorting is correct. For example, if it's sorted by Highest oracle price, then the firstToken's price >= lastToken's price

Reply:
Oh and actually, it would also be better if we could move the sorting e2e to another context, perhaps "Wallet - Loans", since currently it's inside "Wallet - Loans - Take Loans"